src/pks/surface_balance/constitutive_relations/land_cover/seb_threecomponent_evaluator.cc

```diff
@@ -204,6 +203,8 @@ SEBThreeComponentEvaluator::Evaluate_(const State& S,
   const auto& poro = *S.Get<CompositeVector>(poro_key_, tag).ViewComponent("cell",false);
   const auto& ss_pres = *S.Get<CompositeVector>(ss_pres_key_, tag).ViewComponent("cell",false);
 
+
```

Comment:
delete blank lines -- 1 blank line in code or when organizing lists of functions, 2 between function definitions only.